src/data/zmod/basic.lean

```diff
@@ -204,52 +204,75 @@ by { cases n; refl }
 
 end
 
-lemma nat_cast_surjective [fact (0 < n)] :
-  function.surjective (coe : ℕ → zmod n) :=
+/-- So-named because the coercion is `nat.cast` into `zmod`. For `nat.cast` into an arbitrary ring,
+see `zmod.nat_cast_val`. -/
+@[simp]
+lemma nat_cast_zmod_val {n : ℕ} [fact (0 < n)] (a : zmod n) :
+  (a.val : zmod n) = a :=
```

Comment:
```suggestion
lemma nat_cast_zmod_val {n : ℕ} [fact (0 < n)] (a : zmod n) : (a.val : zmod n) = a :=
```

Reply:
It looks like one of the original authors of this file preferred the wrapped style, so I was trying to be consistent. It looks like the rest of the file isn't consistent anyway, so I'll commit your suggestions.